backend/src/test/java/de/pawcheck/backend/cat/CatServiceTest.java

```diff
@@ -13,10 +19,26 @@
 
 class CatServiceTest {
 
-    CatRepo catRepo = mock(CatRepo.class);
-    IdService idService = mock(IdService.class);
-    UserRepo userRepo = mock(UserRepo.class);
-    CatService catService = new CatService(catRepo,idService,userRepo);
+    @Mock
+    private CatRepo catRepo;
+    @Mock
+    private UserRepo userRepo;
+    @Mock
+    private IdService idService;
+    @Mock
+    private MongoTemplate mongoTemplate;
```

Comment:
Diese Zeilen zusammen mit dem `MockitoAnnotations.initMocks(this);` in Zeile 40 machen das Mocking ja schön kompakt. Werde ich mir auch abkucken.